src/overview/components/ResultList.jsx

```diff
@@ -27,6 +27,15 @@ const timeGapToSpaceGap = makeNonlinearTransform({
 })
 
 const ResultList = ({searchResult}) => {
+
+    //if no result found display relavant message
+    if(searchResult.rows.length === 0){
+        return <p className={styles.message}>
```

Comment:
only about indentation, and only if you think my way is better, and only if my way doesn't violate the rest of the project. To show another big project on this style https://github.com/acdlite/redux-router/blob/master/examples/basic/index.js#L49 (line 49)
I normally don't put weight on style but I started like that with the return and it made my life much harder when the project grew